### ISSUE TITLE
mapred regression test failed with Python 2.7

### ISSUE THREAD

**Issue by yydzero** [greenplum-db/gpdb#89]
```
○ → cat regression.diffs 
*** ./expected/mapred.out   2015-11-16 18:03:18.068985485 +0800
--- ./results/mapred.out    2015-11-16 18:03:18.096984211 +0800
***************
*** 97,104 ****
  $$ language plpythonu NO SQL;
  SELECT python_version() FROM env GROUP BY python_version;
      python_version     
! -----------------------
!  (2, 6, 2, 'final', 0)
  (1 row)

  --
--- 98,105 ----
  $$ language plpythonu NO SQL;
  SELECT python_version() FROM env GROUP BY python_version;
                                 python_version                                
! -----------------------------------------------------------------------------
!  sys.version_info(major=2, minor=7, micro=5, releaselevel='final', serial=0)
  (1 row)

  --

======================================================================
```

**Comment by ivannovick** [comment]
@danielgustafsson I know you love gpmapreduce?  Is this issue valid?

**Comment by danielgustafsson** [comment]
Who doesn't love gpmapreduce? 😄 This was fixed in 595586bbe654cd538144298f1e0b2f63c101fd22 which moved the version checks (for Python and Perl) in under ignore blocks. Closing this as it's a solved issue.
